.github/workflows/release.yml

```diff
@@ -13,7 +13,29 @@ permissions:
   contents: read
   id-token: write
 jobs:
+  tag:
+    runs-on: ubuntu-latest
+    needs: [build]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Tag
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          git tag -fa canary -m "Latest Continuous Release" ${GITHUB_SHA}
+          git push --force origin canary
```

Comment:
Is this worth a `--force-with-lease` just in case? Maybe also just push the canary tag and nothing else?